src/libs/dbHandler.go

```diff
@@ -26,6 +26,12 @@ func GetNetworkPolicies(cfg types.ConfigDB, cluster, namespace, status string) [
 			return results
 		}
 		results = docs
+	} else if cfg.DBDriver == "sqlite3" {
```

Comment:
Lets just use `sqlite` as the driver name.

Reply:
If we don't specify `sqlite3` then it doesn't identify the sqlite driver

Reply:
The driver name should match the name as per the driver value in local-file config file. So sqlite should work.

Reply:
Yeah so I tried with `sqlite` and it doesn't recognise the driver, also in the local-file the name of the driver is `sqlite3` , because anything else and it doesn't recognise the driver